src/components/Button.tsx

```diff
@@ -71,18 +74,27 @@ export default React.memo(function Button(props: ButtonProps) {
         <Touchable
           onPress={debouncedOnPress}
           disabled={disabled}
-          style={getStyle(size, backgroundColor, opacity)}
+          style={[getStyle(size, backgroundColor, opacity, iconPositionLeft)]}
```

Comment:
nit: array is not necessary
```suggestion
          style={getStyle(size, backgroundColor, opacity, iconPositionLeft)}
```

Reply:
ah yes, thank you! forgot this was there